CONTRIBUTING.md

```diff
@@ -233,10 +233,10 @@ A pull request should consist of commits with messages that clearly state what p
 
 Commit logs should be stated in the active, present tense.
 
-A commit's subject should be 72 characters or less.  Overall, think of
+A commit's subject should be 72 characters or fewer. Overall, think of
```

Comment:
Nobody says, "n or fewer." If the usual idiom is too casual: "The subject line of a commit message should be no more than 72 characters."

Reply:
Thank you. I have corrected it. Am I misunderstanding this?https://www.grammarly.com/blog/fewer-vs-less/

Reply:
You get the rule, but the VC on the elevator would not say: "Give me your pitch in ten words or fewer." It's just an idiom. I read the link. Maybe it's because fewer implies (unwarranted) precision. "Under" also serves: "under 10 words", but it's like `(1 to 72)`, "under 72 chars". I never remember `to` versus `until` except that `1 to 10` is what I would expect.

"Should I eat a hundred jellybeans before exercising or fewer than that?" Maybe the extra words help coordinate the grammar.

Reply:
Great! Thanks!